source/_integrations/thread.markdown

```diff
@@ -16,12 +16,30 @@ ha_zeroconf: true
 
 The Thread integration helps you track the different Thread networks in your home and helps you manage their credentials. It is currently a work in progress.
 
-Thread is a low-power mesh networking standard which allows users to connect their devices within a home network. It uses the same RF technology as Zigbee, but provides IP connectivity similar to Wi-Fi. Unlike Zigbee, Thread does not allow to control devices directly: It is just a communication protocol. A higher level protocol such as Matter or HomeKit is required to control Thread enabled devices. Thread allows devices to communicate with each other without the need for a central gateway or router. This makes it ideal for home automation, where a large number of devices may need to be connected.
+##### A networking standard and communication protocol - not a device control standard
+Thread is a low-power mesh networking standard which allows users to connect their devices within a home network. Thread allows devices to communicate with each other without the need for a central controller. This makes it ideal for home automation, where a large number of devices may need to be connected. Thread uses the same RF technology as Zigbee, but provides IP connectivity similar to Wi-Fi. Unlike Zigbee, Thread does not allow to control devices directly: It is just a communication protocol. A higher level protocol such as Matter or HomeKit is required to control Thread enabled devices. To see which home automation standard is supported by your device, check the icon on the packaging:
 
+<table>
+  <tr>
+    <th style="width:20%">Logo</th>
+    <th style="width:80%">Supported Home Automation standard</th>
+  </tr>
+  <tr>
+    <td> <img src="https://brands.home-assistant.io/_/matter/icon.png"  width="50"></td>
+    <td><a href="/integrations/matter/">Matter</a></td>
+  </tr>
+  <tr>
+    <td><img src="https://brands.home-assistant.io/_/homekit/icon.png"  width="50"></td>
+    <td><a href="/integrations/homekit/">HomeKit</a></td>
+  </tr>
+</table>
```

Comment:
We should avoid the use of HTML in Markdown as much as possible.

Tables are available in Markdown.

Reply:
Unfortunately, I was not able to resize the columns using markdown. The method using the colon did not work for me in the preview. As a workaround, I shortened the title so that it fits into the 50% column.